src/index.html

```diff
@@ -17,6 +22,39 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo">
+        <img
+          src="images/logo.png"
+          alt="header"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              class="is-active"
+              href="apple"
+            >
```

Comment:
The `href` attribute in the anchor tag should start with a '#' symbol when linking to an anchor on the same page. Also, avoid using spaces in the `href` value.